packages/core/stories/spinner/spinner.doc.stories.mdx

```diff
@@ -38,6 +38,31 @@ Set the `size` prop to `"large"` to use the large Spinner.
   <Story id="core-spinner--large" />
 </Canvas>
 
+## Usage
+
+### When to use Spinner
+
+- To indicate that content is loading.
+- To reassure the user that the system is still working.
+- For full screen and widget loading experiences.
+- For any actions that take between 1-9 seconds. For actions that take 10 seconds or more, it’s recommended to use the Progress component instead.
+
+#### Notes
+
+- Large Spinners can be beneficial for full screen loading experiences; while default Spinners can be reserved for widget loading experiences.
+- If only a portion of a page is being updated, place the Spinner component in that part of the page.
+
+### Do
+
+- Use for actions that take between 1-9 seconds to load.
+- Center spinner horizontally and vertically over a page or content that is loading.
+- If only a portion of a page is being updates, place the Spinner in that part of the page.
+
+### Don't
+
+- Don't use for actions that take 10 seconds or more. Use the Progress component instead.
```

Comment:
@sion-docs @mikearildbrown Is there a standard on how to write these statements?

e.g. should this be re-written to "Don't use the Spinner to show determinable progress of a process. Use the Progress component instead."

Reply:
Yes @mikearildbrown, that's correct.